terms.html

```diff
@@ -114,9 +133,10 @@
 The ability of a <a>holder</a> to make fine-grained decisions about what
 information to share.
   </dd>
-  <dt><dfn data-lt="subjects|subject's">subject</dfn></dt>
-  <dd>
-A thing about which <a>claims</a> are made.
+  <dt><dfn data-lt="subjects|subject's|subject(s)">subject</dfn></dt>
+The <a>entity</a> about which a <a>claim</a> is made.
+The common practice of speaking about "the <a>subject</a> of a <a>verifiable credential</a>" implies 
+that all <a>claims</a> in that <a>verifiable credential</a> are all about precisely one <a>entity</a>.
```

Comment:
I do not believe it is the case that "all claims in a verifiable credential are all about precisely one entity."
A single verifiable credential may have multiple subjects.

Reply:
technically, every instance of `id` is what the claims are bound too (and at each layer of the hierarchy)... so beyond `subject`... claims are bound to the graph node that contains them.

In this example:

```json
{ 
 id: credential-id, 
 name: CoolDegree
 issuer: {
   id: issuer-id
   name: Alice,
 },
 subject: [{
   id: subject-id-1,
   name: Bob
 }, {
   id: subject-id-2,
   name: Charlie
 }]
}
```

You get the following RDF triples (the true data model of W3C Verifiable Credentials)

`credential-id` `name` `CoolDegree`
`credential-id` `issuer` `issuer-id`
`issuer-id` `name` `Alice`
`credential-id` `subject` `subject-id-1`
`subject-id-1` `name` `Bob`
`credential-id` `subject` `subject-id-2`
`subject-id-2` `name` `Charlie`

... same applies to Verifiable Presentations....

Reply:
@brentzundel: your statement is correct, but the proposed text was within a very specific context, which is where one is talking about *the subject* (as opposed to 'some subject') of/in a VC. In that context, where there is only one subject, the implication would be that either all claims in the credential are about that subject, or the speaker is being incoherent.

Reply:
@OR13: yes. And I would like to see that people would acknowledge this by not using phrases such as "the subject of a credential". And while this may seem to be nit-picking, I would argue that using language sloppily is a major source of needless discussions as well as errors in implementations, and should be avoided.

Reply:
My concern with this sentence is that it does not include any indication that to do so is incorrect. 
This suggestion alleviates my concern, but the whole sentence still feels awkward:
```suggestion
The common practice of speaking about "the <a>subject</a> of a <a>verifiable credential</a>" implies 
that all <a>claims</a> in that <a>verifiable credential</a> are all about precisely one <a>entity</a>, which might not be correct.
```

Reply:
Better?